skops/card/tests/test_card.py

```diff
@@ -405,6 +428,93 @@ def test_with_metadata(self, card: Card, meth):
         assert result == expected
 
 
+class TestModelCardFromPath:
+    @pytest.mark.parametrize("file_type", [str, Path])
+    @pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+    def test_model_card_str(self, suffix, file_type):
+        model0 = LinearRegression(n_jobs=123)
+        save_file = save_model_to_file(model0, suffix)
+        save_file = file_type(save_file)
+        print(type(save_file))
```

Comment:
Please remove :)

Reply:
oh, sorry left it by mistake, 
one more thing @BenjaminBossan I'm unable to understand where should i add the changes in `changes.rst` it will go under `v0.3` right.